mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -2445,12 +2446,12 @@ pub async fn taker_swap_trade_preimage(
     let dex_amount = dex_fee_amount_from_taker_coin(my_coin.deref(), other_coin_ticker, &my_coin_volume);
     let taker_fee = TradeFee {
         coin: my_coin_ticker.to_owned(),
-        amount: dex_amount.clone(),
+        amount: dex_amount.fee_amount(),
```

Comment:
We should either show the `total_spend_amount` in the trade preimage response or both the `fee_amount` and the `burn_amount`.  I think we can use `total_spend_amount` for now, if GUIs want to show the burn amount before doing the trade we can update this later to use `DexFee`.